.zenodo.json

```diff
@@ -17,6 +17,11 @@
       "name": "Doerr, Mark",
       "orcid": "0000-0003-3270-6895",
       "affiliation": "Department of Biotechnology and Enzyme Catalysis, Institute of Biochemistry, University of Greifswald, Felix-Hausdorff-Str. 4, 17487 Greifswald, Germany"
+    },
+    {
+      "name": "Schumann, Julia",
+      "orcid": "0000-0002-4041-0165",
+      "affiliation": "Consortium FAIRmat, c/o, Humboldt-Universität zu Berlin, Zum Großen Windkanal 2, 12489 Berlin, Germany"
```

Comment:
"affiliation": "Physics Department and CSMB, Humboldt-Universität zu Berlin, Zum Großen Windkanal 2, 12489 Berlin, Germany"

Reply:
Thanks!